src/ui/hexeditwidget.cc

```diff
@@ -265,5 +332,22 @@ void HexEditWidget::newBinData() {
   visualisation_act_->setEnabled(data_model_->binData().size() > 0);
 }
 
+void HexEditWidget::selectionChanged(qint64 start_addr,
+    qint64 selection_size) {
+  selection_label_->setText(
```

Comment:
I think that one of assumptions we made about this label was that user should be able to easily select it and copy its content (this wasn't true for previous version as well, however we might want to change it now).